### DISCUSSION TITLE
Multiple border routers

### DISCUSSION THREAD

**Discussion by fgervais** [openthread/openthread#7649] [Q&A] [answered]
I will be adding a second border router to my network wired to the same ipv4 network.

Do I need to change anything in the setup compared to my first one?

Do I keep the same network key, mesh prefix  and everything?

**Comment by jwhui** [comment, marked as answer]
> Do I need to change anything in the setup compared to my first one?

No, you should not need to change anything. Thread is designed to support multiple border routers attached to the same Thread network.

> Do I keep the same network key, mesh prefix and everything?

Yes, all Thread devices in the same Thread network should have the same Active Operational Dataset. Note, however, that OpenThread does not require knowing the entire Active Operational Dataset to successfully attach a device to a given Thread network. See [src/cli/README_DATASET.md](https://github.com/openthread/openthread/blob/main/src/cli/README_DATASET.md) for more details.
